js/ui/codeView.js

```diff
@@ -96,8 +98,16 @@ const _ensureHackDataFile = (function () {
             return keyfile;
 
         try {
-            keyfile.load_from_dirs('hack-data.ini', searchPaths,
+            const [, path] = keyfile.load_from_dirs('hack-data.ini', searchPaths,
```

Comment:
Technically you need to monitor all the paths in `searchPaths`, otherwise the config could originally be loaded from a directory low down in the priority order, then overridden by one in a different (higher priority) directory.

Reply:
We're not reloading the keyfile, this path is calculated the first time, so if we monitor all the paths we should recalculate this path on every change.

That will be a bit harder to implement. In that case maybe it's easier to monitor all files and when something changes set the keyfile to null and disconnect all monitors so the next time it's requested it'll be calculated.

Reply:
I've changed the code to monitor all files.